en/getting-to-production.html

```diff
@@ -9,62 +9,64 @@
   This guide assumes you have deployed an application to the <code>dev</code>
   zone as detailed in <a href="getting-started">getting started</a>.
 </p>
+<p>Prerequisites:</p>
+<ul>
+  <li>git</li>
+  <li><a href="https://blog.vespa.ai/introducing-vespa-cli/">Vespa CLI</a> or openssl</li>
```

Comment:
```suggestion
  <li><a href="https://docs.vespa.ai/en/vespa-cli/">Vespa CLI</a> or openssl</li>
```
Link direcctly to CLI